src/integrations/admin/addon-installation/installation-integration.php

```diff
@@ -112,10 +119,12 @@ public function install_and_activate_addons() {
 			echo '</p>';
 		}
 
+		$activation_success_url = ( $this->addons_activated ) ? '&activation_success=true' : '';
+
 		printf(
 			/* translators: %1$s expands to an anchor tag to the admin premium page, %2$s expands to Yoast SEO Premium, %3$s expands to a closing anchor tag */
 			esc_html__( '%1$s Continue to %2$s%3$s', 'wordpress-seo' ),
-			'<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_licenses' ) ) . '">',
+			'<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_licenses' . $activation_success_url ) ) . '">',
```

Comment:
Would prefer to use `add_query_arg` to add the additional `activation_success` to the URL.

See https://developer.wordpress.org/reference/functions/add_query_arg/